templates/unminified/cache.tpl.html

```diff
@@ -0,0 +1,4 @@
+<!doctype html>
+<html manifest="manifest.appcache">
+<script defer async src="cache-iframe.js"></script>
+<title>c</title>
```

Comment:
This file looks weird but is actually valid HTML5, as per the [HTML5 Validator](http://html5.validator.nu).